src/NuGet.Clients/NuGet.VisualStudio.Common/EnvDTEProjectInfoUtility.cs

```diff
@@ -24,6 +24,12 @@ public static class EnvDTEProjectInfoUtility
 
         public const string WebConfig = "web.config";
         public const string AppConfig = "app.config";
+        public const string FullPath = "FullPath";
```

Comment:
This doesn't need to be but TargetPlatformMinVersion does need to be to reuse above to get TPMin version from envDTEProject. Ideally I would have liked these values to be consts defined in a separate class or something, but I am avoiding to make any unneeded refactors for now.